cpp/include/rapids_triton/triton/triton_memory_resource.hpp

```diff
@@ -38,7 +38,6 @@ struct triton_memory_resource final : public rmm::mr::device_memory_resource {
   {
   }
 
-  bool supports_streams() const noexcept override { return false; }
```

Comment:
Do we have warnings as errors on to catch these?

Reply:
We do. I'm pretty sure that this is just a straight-up compilation error rather than just a warning, though.